output-management/web-viewer/README.md

```diff
@@ -10,3 +10,6 @@ Currently available samples:
 
 - [Downloader Application](downloader) - A sample that shows how the REST API can be used to
   implement a batch downloader for text reports into a merged local file.
+
+- [Find Failed Jobs](find-failed-jobs) - A bash script leveraging the View plug-in for Zowe CLI to
```

Comment:
```suggestion
- [Find Failed Jobs](find-failed-jobs) - A bash script leveraging the View Plug-in for Zowe CLI to
```
consistency with [official naming](https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-view-plug-in-for-zowe-cli.html) that is already used in the other README file, possibly also applies to several places within the script itself